contentcuration/contentcuration/urls.py

```diff
@@ -92,7 +92,8 @@ class AssessmentItemViewSet(BulkModelViewSet):
     url(r'^file_upload/', views.file_upload, name="file_upload"),
     url(r'^accounts/logout/$', auth_views.logout, {'template_name': 'registration/logout.html'}),
     url(r'^accounts/', include('django.contrib.auth.urls')),
-    url(r'^channel_edit/', views.edit, name='edit')
+    url(r'^channel_edit/', views.channel_edit, name='channel_edit'),
```

Comment:
Isn't 'creating' just a special version of 'editing'? I think we've had this conversation before.